packages/lambda-tiler/src/util/validate.ts

```diff
@@ -90,17 +88,22 @@ export const Validate = {
     req.set('tileMatrix', tileMatrix.identifier);
     req.set('projection', tileMatrix.projection.code);
 
-    const tileType = Validate.getTileFormat(req.params.tileType);
-    if (tileType == null) throw new LambdaHttpResponse(404, 'Tile extension not found');
-    req.set('extension', tileType);
+    if (req.params.tileType == null) throw new LambdaHttpResponse(404, 'Tile extension not found');
+
+    // trim ".webp" to "webp" and "-terrain-rgb.webp" to "terrain-rgb"
```

Comment:
`"-terrain-rgb.webp" to "terrain-rgb.webp"` ?

Reply:
Yes, If you added this as a suggestion I could have just merged it :)

Reply:
```suggestion
    // trim ".webp" to "webp" and "-terrain-rgb.webp" to "terrain-rgb.webp"
```